setup.py

```diff
@@ -50,7 +50,7 @@
     "docs": ["furo", "sphinx", "zope.interface", "sphinx-notfound-page"],
     "tests_no_zope": [
         # For regression test to ensure cloudpickle compat doesn't break.
-        "cloudpickle",
+        'cloudpickle; python_implementation != "PyPy"',
```

Comment:
Since cloudpickle depends upon the CPython internals, maybe `python_implementation == "CPython"` instead? (I am not familiar enough with packaging to know exactly what the implementation name and syntax should be; but whatever means “install this only on CPython”, rather than just “install this unless it’s PyPy”.)

Reply:
Sure, done that and squashed the commits.